src/encoded/types/dataset.py

```diff
@@ -29,7 +29,7 @@
 
 
 def item_is_revoked(request, path):
-    return request.embed(path, '@@object').get('status') == 'revoked'
+    return request.embed(path, '@@object?skip_calculated=true').get('status') == 'revoked'
```

Comment:
explicitly adding the parameter skip_calculated using ? doesn't seem 100% kosher.  Like I feel it would be cleaner if parameters could be passed to request.embed directly?

Reply:
In python 3.5 you can use multiple sets of kwargs.  That is how I would set this up.  request embed would take a dict of **query_args and a dict of **options.  so skip_calc would be a query arg and the is_user would be an option.